5.machine_learning/scripts/3.extract_model_coef.py

```diff
@@ -156,3 +156,22 @@
 # Show the top ten ranking features for predicting "Failing" class
 coefficients_failing_df.head(10)
 
+
+# ## Add ranking column with sorted descending values and save the CSV for visualization
+
+# In[10]:
+
+
+# Sort coefficients_df by descending order
+coefficients_df = coefficients_df.sort_values(by='Coefficient', ascending=False)
+
+# Add a new column 'Rank'
+coefficients_df['Rank'] = range(1, len(coefficients_df) + 1)
```

Comment:
This is minor, so feel free to ignore. Consider adding a noun to the column name here to help specify what you're ranking here. Including it within the data will help make the code and the data more clear.

Reply:
Not 100% what noun to include so I will keep the rank name but add a note in the subtitle for the code block regarding how the rank works more.